### DISCUSSION TITLE
Quick Tags on Queue page

### DISCUSSION THREAD

**Discussion by lupomikti** [AprilSylph/XKit-Rewritten#277] [Help Desk] [answered]
I didn't see any issues about this so I thought I'd ask here first in case I missed something. Whenever I try to use Quick Tags while in my queue I get an error message saying "Posting failed. Please try again." If Quick Tags is no longer meant to work in the queue, is it possible to not have the icon on the queued post? Or is this an actual issue that needs to be resolved? [Browser version, if applicable: Firefox 92.0 (20210903235534); using latest 0.8.2 release].

**Comment by AprilSylph** [comment, marked as answer]
This is a problem I was unaware of. It looks like the post `state` that Tumblr gives us from queued post is `queued`, while it wants us to send `queue` to modify the post.

**Reply by AprilSylph** [reply]
Acknowledged as Quick Tags / Trim Reblogs issue at #278

**Reply by lupomikti** [reply]
I see! Thanks for the info.

**Reply by AprilSylph** [reply]
Tumblr has now fixed this at my request. Let me know if it breaks again!

**Reply by lupomikti** [reply]
Awesome! Glad that was resolved so quickly. Confirmed on my end as well that the feature is properly working now. Thank you!

**Reply by lupomikti** [reply]
@AprilSylph I figured it's better to ask such a quick question here than open a whole issue for it right off the bat. Is there anything that prevents Quick Tags from recognizing a scheduled post from a normal queued post? I ask because adding tags to a scheduled post (w/o opening the post editor, to be clear) does not save the post with the same schedule and instead adds it into the normal auto-queue.

**Reply by AprilSylph** [reply]
Quick Tags needs to be updated to handle the `publish_on` variable in order to preserve manually-set schedule times. Thanks for catching this, I'll get on it as soon as I can!
